### ISSUE TITLE
Altitude Range Arc (ARC) - AKA Green Banana not working

### ISSUE THREAD

**Issue by Tinpanva** [flybywiresim/a32nx#4958]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-05-30T15:02:19+00:00",
  "ref": "master",
  "sha": "90cfea496cae7a14eb3540b2f1a8bb7a06d33181",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

The ARC is no longer showing up on my 3rd party EFB software (Aivlasoft) but continues to still work in the stable version. The so-called "Green Banana" depicts the approximate position where a climb or descent to
a preset altitude will end.

### Expected behavior

On decent I will use the ARC to accurately cross a waypoint at the correct height. 

### Steps to reproduce

The aircraft must be equipped with an autopilot
including Altitude Preselect function
• The aircraft must be airborne
• The autopilot must be ON
• The option ARC must be activated
• At least one adjustment of the Altitude Preselect
function must be executed. it the Altitude Preselect is
set to a certain altitude already at Simulator startup,
the altitude is NOT recognized. A change in Altitude
Preselect is required.
• The option "use vertical

**Comment by Watsi01** [comment]
The A320 has no green banana.

**Comment by Tinpanva** [comment]
As stated in the report, "while using the Aivlasoft software". That
software produces a green banana. Still does in the stable version, It used
to in the dev version. I completely understand that the green banana is a
Boeing perk,  but the EFB software reproduces it for any aircraft.

**Comment by tracernz** [comment]
The A320 has an "energy circle" symbol on the ND, which is an arc showing the distance required to land when in descent and approach phases with selected heading or track engaged. That is not yet implemented on our ND (#856), but eventually it will be. Not sure what the Boeing one does.

We don't know how Aviasoft calculates their arc, so it's not really feasible for us to make it work without knowing the sim variables they are using (assuming that's how they do it).

**Comment by MisterChocker** [comment]
Duplicate of #856
